src/utils/hooks/api/xcm/use-xcm-bridge.ts

```diff
@@ -4,6 +4,7 @@ import { firstValueFrom } from 'rxjs';
 
 import { XCM_ADAPTERS } from '@/config/relay-chains';
 import { BITCOIN_NETWORK } from '@/constants';
+import { Chains } from '@/pages/Transfer/CrossChainTransferForm/components/ChainSelect';
```

Comment:
Consider moving this type to `src/types` folder, where is host as well types for `loans` and so on

Reply:
Yep good idea—feel free to make this change when you're integrating the form utilities

Reply:
Actually I'll do it now :)